.circleci/config.yml

```diff
@@ -24,6 +24,11 @@ jobs:
           name: cargo test
           command: |
             cargo test
+      - run:
+          name: cargo clippy
+          command: |
+            cargo clean
+            cargo clippy -- -D warnings
```

Comment:
This could be `cargo clippy -- --allow clippy::module_name_repetitions` if u want to ignore one particular lint rule (in case it's buggy)

Reply:
I don't think it is buggy, see: https://rust-lang.github.io/rust-clippy/master/#module_name_repetitions